### PR TITLE
+ AUTaxFileNumberField

### PR DESCRIPTION
**Pull request by FrancoisConstant**
Hello,

I've added AUTaxFileNumberField (form and model) + AUTaxFileNumberFieldValidator.

It's based on the AUBusinessNumberField.

This is my first contribution and I hope it's all good.

**All Changes**

- [x] Add an entry to the docs/changelog.rst describing the change.

- [x] Add an entry for your name in the docs/authors.rst file if it's not
      already there.

- [x] Adjust your imports to a standard form by running this command:

      `isort --recursive --line-width 120 localflavor tests`

**New Fields Only**

- [x] Prefix the country code to all fields.

- [x] Field names should be easily understood by developers from the target
      localflavor country. This means that English translations are usually
      not the best name unless it's for something standard like postal code,
      tax / VAT ID etc.

- [x] Prefer '<country code>PostalCodeField' for postal codes as it's
      international English; ZipCode is a term specific to the United
      States postal system.

- [x] Add meaningful tests. 100% test coverage is not required but all
      validation edge cases should be covered.

- [x] Add `.. versionadded:: <next-version>` comment markers to new
      localflavors.

- [x] Add documentation for all fields.

### CODE REVIEW

**Comment by benkonrath** [comment]
Thanks for your contribution. I'll be able to review this over the weekend during the Dutch Django sprint.

File: docs/changelog.rst [hunk 1]
```diff
@@ -13,6 +13,8 @@ New fields for existing flavors:
 - Added MXCLABEField model and form fields.
   (`gh-227 <https://github.com/django/django-localflavor/pull/227>`_).
 
+- Added AUTaxFileNumberField model and form fields.
```
Comment (benkonrath) [inline docs/changelog.rst]: Please add a link to the this github issue like the other changelog entries.

**Comment by benkonrath** [comment]
Looks great! Thanks for your contribution. Please update the changelog as requested and rebase / squash all the commits into one commit. Once that this done I'll merge this PR. Thanks.

**Comment by FrancoisConstant** [comment]
Thanks @benkonrath -  the commits are squashed.

**Comment by benkonrath** [comment]
Merged via fd9fcf1fc61d2aeaedccf209ec03322daf620831.
